### PR TITLE
Trying to make the difference between pear flags and app args more clear

### PR DESCRIPTION
**Pull request by freeall**
image

### CODE REVIEW

File: cmd/usage.js [hunk 1]
```diff
@@ -94,29 +94,30 @@ module.exports = ({ fork, length, key }) => {
   `
 
   const run = ansi.bold(cmd + ' run')
-  const runArgs = ansi.bold('<key|dir|alias> [...args]')
-  const runBrief = 'Run an application from a key or dir.'
+  const runArgs = ansi.bold('[flags] <key|dir|alias> [...app-args]')
+  const runBrief = 'Run an application from a key or dir. app-args are passed directly to the app.'
```
Comment (davidmarkclements) [inline cmd/usage.js:98]: this is displayed in main help so it needs to be brief (as per the name) 

You can put this explanation in runExplain though

File: cmd/usage.js [hunk 2]
```diff
@@ -94,29 +94,30 @@ module.exports = ({ fork, length, key }) => {
   `
 
   const run = ansi.bold(cmd + ' run')
-  const runArgs = ansi.bold('<key|dir|alias> [...args]')
-  const runBrief = 'Run an application from a key or dir.'
+  const runArgs = ansi.bold('[flags] <key|dir|alias> [...app-args]')
```
Comment (davidmarkclements) [inline cmd/usage.js:97]: yeah cool ok - can you update `pear dev` also

File: cmd/usage.js [hunk 3]
```diff
@@ -94,29 +94,30 @@ module.exports = ({ fork, length, key }) => {
...
+           --checkout=n               Run a checkout, n is version length
+           --checkout=release         Run checkout from marked released length
+           --checkout=staged          Run checkout from latest version length
     ${ansi.dim(ansi.italic(`
-     pear run pear://u6c6it1hhb5serppr3tghdm96j1gprtesygejzhmhnk5xsse8kmy
-     pear run pear://keet
-     pear run file://path/to/an-app-folder
-     pear run path/to/an-app-folder --some args
-     `))}`
+    pear run pear://u6c6it1hhb5serppr3tghdm96j1gprtesygejzhmhnk5xsse8kmy
```
Comment (davidmarkclements) [inline cmd/usage.js:114]: leave positioning as is

**Review by davidmarkclements — CHANGES_REQUESTED** [review]
.

File: cmd/usage.js (hunk 3 above)
Comment (freeall) [reply cmd/usage.js:114]: Is it on purpose that it's not aligned with the rest?

File: cmd/usage.js [hunk 4]
```diff
@@ -94,29 +94,30 @@ module.exports = ({ fork, length, key }) => {
...
     ${ansi.bold('alias')}  pear://<alias>
-
-    --dev                      Run the app in dev mode
-    --link=url                 Simulate deep-link click open
-    --store|-s=path            Set the Application Storage path
-    --tmp-store|-t             Automatic new tmp folder as store path
-    --checkout=n               Run a checkout, n is version length
-    --checkout=release         Run checkout from marked released length
-    --checkout=staged          Run checkout from latest version length
+    ${ansi.bold('flags')}  --dev                      Run the app in dev mode
```
[the remark this answers is not in the document]
Comment (freeall) [reply cmd/usage.js:106]: image

It was actually to get it aligned with the top ones. Looks like this when printed out.

[the remark this answers is not in the document]
Comment (davidmarkclements) [reply cmd/usage.js:108]: @freeall I mean everything else as in all other commands - remove flags and just keep the flags below 

what we could do is label all flag sections with flags but can do that separately

File: cmd/usage.js (hunk 3 above)
Comment (davidmarkclements) [reply cmd/usage.js:116]: it should be aligned (and currently is)

Screenshot 2024-02-22 at 17 41 03

File: cmd/usage.js [hunk 5]
```diff
@@ -23,7 +23,7 @@ module.exports = ({ fork, length, key }) => {
     --with|-w=name   Additional functionality. Available: node
   `
   const dev = ansi.bold(cmd + ' dev')
-  const devArgs = ansi.bold('[dir] [...args]')
+  const devArgs = ansi.bold('[flags] [dir] [...args]')
```
Comment (davidmarkclements) [inline cmd/usage.js:26]:
```suggestion
  const devArgs = ansi.bold('[flags] [dir] [...app-args]')
```

**Review by davidmarkclements — APPROVED** [review]

**Pull request merged**